tests/e2e/utils/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - `utils.waitForTimeout( delay )` pause processing for `delay` milliseconds
 - `AdminEdit` class with utility functions for the respective edit screens
+- Update `shopper.addToCartFromShopPage()` to accept product Id or Title
```

Comment:
We should also note that `shopper.removeFromCart()` has been updated to accept product ID or title as well.

Reply:
Added in https://github.com/woocommerce/woocommerce/pull/30822/commits/ef778b898599d50e3775bbdc5909146cb89e4bfa